mass_editing/__openerp__.py

```diff
@@ -5,17 +5,19 @@
     'name': 'Mass Editing',
     'version': '9.0.1.0.0',
     'author': 'Serpent Consulting Services Pvt. Ltd., '
+              'brain-tec AG, '
               'Odoo Community Association (OCA)',
     'contributors': [
         'Oihane Crucelaegui <oihanecrucelaegi@gmail.com>',
         'Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>',
+        'Raul Martin <raul.martin@braintec-group.com>',
     ],
     'category': 'Tools',
     'website': 'http://www.serpentcs.com',
     'license': 'GPL-3 or any later version',
     'summary': 'Mass Editing',
     'uninstall_hook': 'uninstall_hook',
-    'depends': ['base'],
+    'depends': ['base', 'mail'],
```

Comment:
Why do you need to depends on mail?

Reply:
Hi, I don't remember exactly. None of my contributions need it. Maybe for this existing test? test_mass_edit_email

Reply:
No, that one is only for changing email field. Please remove the extra dependency to see.